DER/DerivativesContracts/RightsAndWarrants.rdf

```diff
@@ -203,6 +391,78 @@
 		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">The strike price, in the case of a perpetual warrant, is usually higher than the market value of the underlying asset at the time of issue.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-der-drc-raw;PreferredConvertibleSharePurchaseRight">
+		<rdfs:subClassOf rdf:resource="&fibo-der-drc-raw;PurchaseRight"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-drc-bsc;hasUnderlier"/>
+				<owl:someValuesFrom>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;hasIdentity"/>
+						<owl:someValuesFrom rdf:resource="&fibo-sec-eq-eq;ConvertiblePreferredShare"/>
+					</owl:Restriction>
+				</owl:someValuesFrom>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">preferred convertible share purchase right</rdfs:label>
+		<skos:definition xml:lang="en">purchase right that gives a prospective acquiree&apos;s preferred, convertible shareholders the right to buy preferred, convertible shares of the firm or preferred, convertible shares of anyone who acquires the firm at a deep discount to their fair market value</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom xml:lang="en">ISO 10962, Securities and related financial instruments - Classification of financial instruments (CFI) code, Fourth Edition, October 2019</fibo-fnd-utl-av:adaptedFrom>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-drc-raw;PreferredConvertibleShareSubscriptionRight">
+		<rdfs:subClassOf rdf:resource="&fibo-der-drc-raw;SubscriptionRight"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-drc-bsc;hasUnderlier"/>
+				<owl:someValuesFrom>
+					<owl:Restriction>
+						<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;hasIdentity"/>
+						<owl:someValuesFrom rdf:resource="&fibo-sec-eq-eq;ConvertiblePreferredShare"/>
+					</owl:Restriction>
+				</owl:someValuesFrom>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">preferred convertible share subscription right</rdfs:label>
+		<skos:definition xml:lang="en">subscription right entitling existing preferred, convertible shareholders to subscribe to new securities at a price normally lower than the prevailing market price</skos:definition>
```

Comment:
only the specific preferred convertible share, surely - not any "new securities"